src/index.ts

```diff
@@ -92,9 +97,7 @@ export interface UploadConfig {
    * emulator context is detected.
    */
   authClient?: {
-    request: <T = any>(
-      opts: GaxiosOptions
-    ) => Promise<GaxiosResponse<T>> | GaxiosPromise<T>;
+    request: (opts: GaxiosOptions) => Promise<GaxiosResponse> | GaxiosPromise;
```

Comment:
Was this change because it was defined as `any` type?

Reply:
Yep

Reply:
Good point - I reverted this change with a generic `<T>` (without the `any`)